src/components/Navbar/Navbar.js

```diff
@@ -1,42 +1,60 @@
 //Dependencies
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
 
+//Components, assets, actions, styles etc..
+import NavItem from './NavItem';
+import { ReactComponent as HUOALogo } from '../../assets/huoa-logo.svg';
 import './Navbar.scss';
 
-class Navbar extends React.Component {
-  render() {
-    return (
-      <div>
-        <Link to="/">
-          <h4>HUOA Home</h4>
-        </Link>
-        <ul>
-          <Link to="/about">
-            <li>About</li>
-          </Link>
-          <Link to="/join">
-            <li>Get Involved</li>
-          </Link>
-          <Link to="/events">
-            <li>Events</li>
-          </Link>
-          <Link to="/ogsh">
-            <li>OGSH</li>
-          </Link>
-          <Link to="/news">
-            <li>News</li>
-          </Link>
-          <Link to="/contact">
-            <li>Contact</li>
-          </Link>
-          <Link to="/donate">
-            <li>Donate</li>
-          </Link>
-        </ul>
-      </div>
-    );
-  }
-}
+const Navbar = () => {
+  const [ navState, setNavState ] = useState(false);
+  const { routes } = useSelector((state) => state.router);
+
+  const renderMenuItems = (children) => {
+    return children.map((item) => (
+      <Link className="nav--dropdown__link" to={item.linkTo} onClick={() => setNavState(false)} key={item.linkTo}>
+        <li className="nav--dropdown__link--txt">{item.name}</li>
+      </Link>
```

Comment:
Too much nesting with naming convention

why can't we just do this 
```suggestion
      <Link to={item.linkTo} onClick={() => setNavState(false)} key={item.linkTo}>
        <li className="nav--dropdown__link">{item.name}</li>
      </Link>
```